fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_life.dm

```diff
@@ -21,7 +21,7 @@
 	if(my_clan == CLAN_TREMERE)
 		var/area/current_area = get_area(owner.current)
 		if(istype(current_area, /area/service/chapel))
-			to_chat(owner.current, span_warning("You don't belong in holy areas!"))
+			to_chat(owner.current, span_warning("You don't belong in holy areas! The Faith burns you!"))
```

Comment:
'The Faith'???

Reply:
> As part of the Tremere Clan, you are weak to **True Faith**, as such are unable to enter the Chapel.

I stole and shortened the term from the flavor text of Tremere Clan, as well as Gangrel in the Book of the Kindred. For some reason, the Book of the Kindred does not contain mention of the Tremere clan weakness to being in the Chapel.

Reply:
To an extent, it is "The Faith" because Christianity is canon is VTM, so there's inevitably going to be some conflicts with SS13 lore. I think the ambiguity of "The Faith" makes it fine